ipn/ipnlocal/local.go

```diff
@@ -156,30 +156,31 @@ type watchSession struct {
 // state machine generates events back out to zero or more components.
 type LocalBackend struct {
 	// Elements that are thread-safe or constant after construction.
-	ctx                   context.Context    // canceled by Close
-	ctxCancel             context.CancelFunc // cancels ctx
-	logf                  logger.Logf        // general logging
-	keyLogf               logger.Logf        // for printing list of peers on change
-	statsLogf             logger.Logf        // for printing peers stats on change
-	sys                   *tsd.System
-	e                     wgengine.Engine // non-nil; TODO(bradfitz): remove; use sys
-	store                 ipn.StateStore  // non-nil; TODO(bradfitz): remove; use sys
-	dialer                *tsdial.Dialer  // non-nil; TODO(bradfitz): remove; use sys
-	pushDeviceToken       syncs.AtomicValue[string]
-	backendLogID          logid.PublicID
-	unregisterNetMon      func()
-	unregisterHealthWatch func()
-	portpoll              *portlist.Poller // may be nil
-	portpollOnce          sync.Once        // guards starting readPoller
-	gotPortPollRes        chan struct{}    // closed upon first readPoller result
-	varRoot               string           // or empty if SetVarRoot never called
-	logFlushFunc          func()           // or nil if SetLogFlusher wasn't called
-	em                    *expiryManager   // non-nil
-	sshAtomicBool         atomic.Bool
-	webClientAtomicBool   atomic.Bool
-	shutdownCalled        bool // if Shutdown has been called
-	debugSink             *capture.Sink
-	sockstatLogger        *sockstatlog.Logger
+	ctx                       context.Context    // canceled by Close
+	ctxCancel                 context.CancelFunc // cancels ctx
+	logf                      logger.Logf        // general logging
+	keyLogf                   logger.Logf        // for printing list of peers on change
+	statsLogf                 logger.Logf        // for printing peers stats on change
+	sys                       *tsd.System
+	e                         wgengine.Engine // non-nil; TODO(bradfitz): remove; use sys
+	store                     ipn.StateStore  // non-nil; TODO(bradfitz): remove; use sys
+	dialer                    *tsdial.Dialer  // non-nil; TODO(bradfitz): remove; use sys
+	pushDeviceToken           syncs.AtomicValue[string]
+	backendLogID              logid.PublicID
+	unregisterNetMon          func()
+	unregisterHealthWatch     func()
+	portpoll                  *portlist.Poller // may be nil
+	portpollOnce              sync.Once        // guards starting readPoller
+	gotPortPollRes            chan struct{}    // closed upon first readPoller result
+	varRoot                   string           // or empty if SetVarRoot never called
+	logFlushFunc              func()           // or nil if SetLogFlusher wasn't called
+	em                        *expiryManager   // non-nil
+	sshAtomicBool             atomic.Bool
+	webClientAtomicBool       atomic.Bool
+	exposeWebClientAtomicBool atomic.Bool
```

Comment:
Have used `expose` here and elsewhere, but not sure if this is the best name / phrasing for this. Open to other suggestions if people have opinions.

Reply:
seems like a reasonable name, but worth adding a short comment here for what it means.